maven-setup/action.yml

```diff
@@ -3,6 +3,21 @@ description: Setup action to capture Maven Build Scans metadata
 author: Gradle team
 
 inputs:
+  develocity-injection-enabled:
```

Comment:
It is not obvious but the inputs are sorted by (imo) order of importance. I feel the DV injection is not  the main use case served by this action, would you mind moving these inputs just before the `github-token`?

Reply:
Sure, I rearranged them